docker-compose.yml

```diff
@@ -68,8 +57,9 @@ services:
     volumes:
       - ./nginx/nginx.conf.template:/etc/nginx/nginx.conf.template
     environment:
-      - PROXY_PASS_URL=http://s3-helper:8080/
-      - AUTH_URL=http://avalon:3000
+      - AVALON_DOMAIN=http://avalon:3000
+    env_file:
+      - .env
```

Comment:
FWIW I had `- AVALON_STREAMING_BUCKET=${AVALON_STREAMING_BUCKET}`.  I thought previously env vars had to be referenced in `environment` in order for them to be exposed from the `env_file` to the container, but maybe that has changed?  Either approach seems fine to me but I might prefer being explicit unless there is a desire to be able to use more env vars in the nginx config.

Reply:
Ah! Good catch. I'd much prefer being explicit on what variables it needs. I amended my commit accordingly. (I tested it, and we can even just give the bare variable.)